set.mm

```diff
@@ -532760,6 +532528,73 @@ dependency is expressed in our hypothesis (called implicit
       CDBJABCDKL $.
   $}
 
+  ${
+    $d t x $.  $d x ph $.
+    $( Reverting substitution yields the original expression.  (Contributed by
+       Wolf Lammen, 5-Aug-2023.) $)
+    wl-ich-lem1 $p |- ( [ t / x ] [ x / t ] ph <-> ph ) $=
+      ( wsb sbco2vv sbid bitri ) ACBDBCDACCDAACCBEACFG $.
+  $}
+
+  ${
+    $d a x $.  $d a b y $.  $d a b ph $.
+    $( Reverting two substitutions in succession.  (Contributed by Wolf Lammen,
+       5-Aug-2023.) $)
+    wl-ich-lem2 $p |- ( [ y / b ] [ x / a ] [ a / x ] [ b / y ] ph <-> ph ) $=
+      ( wsb wl-ich-lem1 sbbii bitri ) ACEFZBDFDBFZECFJECFAKJECJDBGHAECGI $.
+  $}
+
+  $( ~ spsbim applied twice.  (Contributed by Wolf Lammen, 5-Aug-2023.) $)
+  wl-ich-lem3 $p |- ( A. a A. b ( ph <-> ps ) ->
```

Comment:
It is ~spsbbi applied twice, isn't it? Why not use ~spsbbi (twice) within the proof?

Reply:
You're right.